spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/support/TaskSqlServerSequenceMaxValueIncrementer.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.task.repository.support;
+
+import javax.sql.DataSource;
+
+import org.springframework.jdbc.support.incrementer.AbstractSequenceMaxValueIncrementer;
+
+/**
+ * Incrementer using SQL Server's sequence.
+ * @author Glenn Renfro
+ * @since 2.3.2
+ */
+public class TaskSqlServerSequenceMaxValueIncrementer extends AbstractSequenceMaxValueIncrementer {
```

Comment:
See previous note about removing the `Task` prefix from the name of this class. Also...unit test for this class?